src/bin/travis/resources/centaur.inputs

```diff
@@ -1,7 +1,7 @@
 {
-  "centaur_workflow.centaur.cromwell_jar":"gs://cloud-cromwell-dev/travis-centaur/CROMWELL_JAR",
-  "centaur_workflow.centaur.centaur_branch":"CENTAUR_BRANCH",
-  "centaur_workflow.centaur.conf":"gs://cloud-cromwell-dev/travis-centaur/multiBackend.conf",
+  "centaur_workflow.centaur.cromwell_jar":"gs://cloud-cromwell-dev/travis-centaur/cromwell-26-1c78b0d-SNAP.jar",
+  "centaur_workflow.centaur.centaur_branch":"develop",
+  "centaur_workflow.centaur.conf":"gs://cloud-cromwell-dev/travis-centaur/multiBackend_updated.conf",
```

Comment:
woops that was very much for my own testing as the new stanza broke the existing config. Removed.